rust/src/lib/route.rs

```diff
@@ -349,6 +338,16 @@ impl RouteEntry {
         self.destination.as_ref().map(|d| is_ipv6_addr(d.as_str()))
             == Some(true)
     }
+
+    pub(crate) fn is_unicast(&self) -> bool {
+        let route_type = [
+            RouteType::Blackhole,
+            RouteType::Unreachable,
+            RouteType::Prohibit,
+        ];
```

Comment:
No need to generate this array at runtime whenever this function been invoked. Please place it as `const` to `RouteType`.

Reply:
on a second through, why doing deny list here? We should do allowlist. Just check it by `self.route_type == RouteType::Unicast`

Reply:
the current enum of Routetype does not have `RouteType::Unicast`, but we can add support for this.

Reply:
no need to add new. Please use struct level `const`.

Reply:
I updated it into:

```
    pub(crate) fn is_unicast(&self) -> bool {
        self.route_type.is_none()
            || u8::from(self.route_type.unwrap()) == RTN_UNICAST
    }
```